### ISSUE TITLE
WCAG 2.4.4,WCAG 4.1.2: Ensure links have discernible text (.text-decoration-none.text-muted)

### ISSUE THREAD

**Issue by Tjoosten** [Tjoosten/vl-woordenboek#200]
#### Issue
`Ensure links have discernible text` ([link-name](https://accessibilityinsights.io/info-examples/web/link-name))
#### Target application
[VL. woordenboek | Gerief](http://vlaams-woordenboek.test/willekeurig-woordenboek-artikel)
#### Element path
.text-decoration-none.text-muted
#### Snippet
`<a href="http://vlaams-woordenboek.test/" class="text-muted text-decoration-none">`
#### How to fix
    Fix all of the following:
      Element is in tab order and does not have accessible text
    
    Fix any of the following:
      Element does not have text that is visible to screen readers
      aria-label attribute does not exist or is empty
      aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
      Element has no title attribute
#### Environment
Chrome version 136.0.0.0
This accessibility issue was found using Accessibility Insights for Web 2.46.0 (axe-core 4.10.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).
